components/compas-open/src/compas-open.ts

```diff
@@ -24,19 +17,18 @@ export function newDocRetrievedEvent(
   docName: string,
   docId?: string
 ): DocRetrievedEvent {
-  return new CustomEvent<DocRetrievedDetail>('doc-retrieved', {
+  return new CustomEvent<DocRetrievedDetail>("doc-retrieved", {
     bubbles: true,
     composed: true,
     detail: { localFile, doc, docName, docId },
   });
 }
 
-@customElement('compas-open')
 export class CompasOpenElement extends LitElement {
```

Comment:
missing JSDoc here and for every prop like how it's done in `oscd-components` 
https://github.com/openscd/oscd-components/blob/main/components/oscd-action-icon/src/oscd-action-icon.ts

Reply:
Out of scope